### ISSUE TITLE
Will CUDA 10.2 binaries come out soon?

### ISSUE THREAD

**Issue by Laksh1997** [rusty1s/pytorch_sparse#67]
To avoid this error:

```
    f'Detected that PyTorch and torch_sparse were compiled with '
RuntimeError: Detected that PyTorch and torch_sparse were compiled with different CUDA versions. PyTorch has CUDA version 10.2 and torch_sparse has CUDA version 10.1. Please reinstall the torch_sparse that matches your PyTorch install.

```

**Comment by rusty1s** [comment]
They are already available for PyTorch 1.5.

**Comment by Laksh1997** [comment]
How do I access them?
